cypress/integration/all-flaky-tests/mandatee-assigning.spec.js

```diff
@@ -334,18 +334,30 @@ context('Assigning a mandatee to agendaitem or subcase should update linked subc
     cy.clickReverseTab('Definitief');
     cy.get(newsletter.itemContent.printItemProposal).as('proposals');
     cy.get('@proposals').eq(0)
-      .contains('Op voorstel van minister-president Jan Jambon en viceminister-president Hilde Crevits');
+      .scrollIntoView()
+      .contains('Op voorstel van minister-president Jan Jambon en viceminister-president Hilde Crevits')
+      .should('be.visible');
```

Comment:
This test was failing for me because (from what I could tell), the item with the `Op voorstel van ...` string was not visible, which is why I added the `scrollIntoView` & `should('be.visible')`. Although the two probably aren't needed together.